back/internal/log/log.go

```diff
@@ -0,0 +1,37 @@
+package log
+
+import (
+	"io"
+	"log"
+)
+
+type Output struct {
+	Out    io.Writer
+	Prefix string
+	Flag   int
```

Comment:
I'm not entirely opposed to this flag config, but I suggest that we leave it out for now and add it in as a separate issue if we hit the need. The main reason is that this is silently exposing log package specifics without being obvious, but I also don't prefer pre-formatted formatting. Feel free to leave it in since it's not actually disrupting anything.